public/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <!-- <link rel="stylesheet" href="/style.css"> -->
+  <title>FEC Demo - BLITVA-Amazon</title>
+</head>
+
+<body>
+  <h1>Amazon</h1>
+  <div id="photos"></div>
+  <div id="CustomerQuestions"></div>
+  <div id="CustomerReviews"></div>
+  <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+  <script src="http://localhost:4002/photos_bundle.js"></script>
+  <script src="http://localhost:4001/customerQuestions_bundle.js"></script>
+  <script src="http://localhost:4006/reviews_bundle.js"></script>
+  <script>
+    // ReactDOM.render(
```

Comment:
You can get rid of the commented out code.

Reply:
Yes, I am planning to do so after discussing few question on this with TM.